mount-helper/install/install.sh

```diff
@@ -238,14 +238,18 @@ _install_apps() {
             fi
             PACKAGE_DIR="packages/ubuntu/$VERSION"
             dpkg --force-all -i "$PACKAGE_DIR/$app"*
-        elif  is_linux LINUX_RED_HAT && ([[ $VERSION == 8.8 || $VERSION == 8.9 ]]); then
+        elif  is_linux LINUX_RED_HAT && ([[ $VERSION == 8.8 || $VERSION == 8.9 || $VERSION == 8.10 ]]); then
             # Read preInstalled packages from system
             if [[ $app != *"python"* && $(grep -q "^$app" $INSTALLED_PACKAGE_LIST; echo $?) -eq 0 ]] ; then
                 log "Skipping package $app for installation as it is pre-installed on the system"
                 continue 
             fi
             log "Installing package $app"
-            if [[ $app == "mount.ibmshare"* || $app == *"python"* ]]; then
+            if [[ $app == "mount.ibmshare"* ]]; then
+                rpm -i "$app" --force --nodeps --nodigest --nosignature
```

Comment:
Why is `--nodigest --nosignature` needed only for mount.ibmshare?

Reply:
In the recent versions, we are seeing gpg issue:
```
Package mount.ibmshare-0.0.4.rpm is not signed
Error: GPG check FAILED
Install failed: Problem installing package: mount.ibmshare-0.0.4.rpm ExitCode(1)
```
to resolve this , we are using no gpg check

Reply:
Got it!